packages/forms-web-app/src/validators/common/email-address.js

```diff
@@ -22,9 +22,6 @@ const ruleYourEmail = (key) =>
 		.isEmail()
 		.withMessage('Enter an email address in the correct format, like name@example.com')
 		.bail()
-		.matches(/^(?=[\w\s])\s*[-+.'\w]*['\w]+@[-.\w]+\.[-.\w]+\s*$/)
```

Comment:
we are already using a library above, so is redundant but this is also hard to maintain, probably doesn't conform to spec and vulnerable to ReDoS